samples/system-test/faceDetection.test.js

```diff
@@ -51,7 +51,7 @@ MockCanvas.Image = class Image {};
 
 const faceDetectionExample = require(`../faceDetection`);
 const inputFile = path.join(__dirname, `../resources`, `face.png`);
-const outputFile = path.join(__dirname, `../../vision`, `out.png`);
```

Comment:
For places where we don't strictly need backticks, can we use single quotes instead please?